usaspending_api/submissions/management/commands/load_dabs_submission_window_schedule.py

```diff
@@ -0,0 +1,112 @@
+
+from usaspending_api.etl.broker_etl_helpers import dictfetchall
+from usaspending_api.submissions.models import DABSSubmissionWindowSchedule
+
+logger = logging.getLogger("console")
+
+# SQL to create Month Period Schedules using broker table
+# Use all periods after Period 9, Year 2020 from table
+# Submission Due Date comes from 'publish_deadline' column
+MONTH_SCHEDULE_SQL = """
+select
+    make_date(year, period, 1) - interval '3 months' as period_start_date,
+    make_date(year, period, 1) - interval '2 months' - interval '1 day' as period_end_date,
+    period_start as submission_start_date,
+    certification_deadline as certification_due_date,
+    publish_deadline as submission_due_date,
+    publish_deadline + interval '1 day' as submission_reveal_date,
+    year as submission_fiscal_year,
+    (period + 2) / 3 as submission_fiscal_quarter,
+    period as submission_fiscal_month,
+    false as is_quarter
+from
+    submission_window_schedule
+where
+    period_start >= '2020-07-17'
+"""
+
+# SQL to create Quarter Period Schedules using broker table
+# Only use periods 3, 6, 9, and 12 from table
+# Submission Due Date comes from 'certification_deadline' column
+QUARTER_SCHEDULE_SQL = """
+select
+    make_date(year, period, 1) - interval '5 months' as period_start_date,
+    make_date(year, period, 1) - interval '2 months' - interval '1 day' as period_end_date,
+    period_start as submission_start_date,
+    certification_deadline as certification_due_date,
+    certification_deadline as submission_due_date,
+    certification_deadline + interval '1 day' as submission_reveal_date,
+    year as submission_fiscal_year,
+    (period + 2) / 3 as submission_fiscal_quarter,
+    period as submission_fiscal_month,
+    true as is_quarter
+from
+    submission_window_schedule
+where
+    period % 3 = 0
+"""
+
+# Manually creating some month period records because they are
+# not represented in the broker table
+EXTRA_MONTH_SCHEDULES = [
+    DABSSubmissionWindowSchedule(
+        period_start_date="2020-04-01 00:00:00",
+        period_end_date="2020-04-30 00:00:00",
+        submission_start_date="2020-07-17 00:00:00",
+        submission_due_date="2020-07-30 00:00:00",
+        certification_due_date="2020-08-14 00:00:00",
+        submission_reveal_date="2020-07-31 00:00:00",
+        submission_fiscal_year=2020,
+        submission_fiscal_quarter=3,
+        submission_fiscal_month=7,
+        is_quarter=False,
+    ),
+    DABSSubmissionWindowSchedule(
+        period_start_date="2020-05-01 00:00:00",
+        period_end_date="2020-05-31 00:00:00",
+        submission_start_date="2020-07-17 00:00:00",
+        submission_due_date="2020-07-30 00:00:00",
+        certification_due_date="2020-08-14 00:00:00",
+        submission_reveal_date="2020-07-31 00:00:00",
+        submission_fiscal_year=2020,
+        submission_fiscal_quarter=3,
+        submission_fiscal_month=8,
+        is_quarter=False,
+    ),
```

Comment:
These are in the broker table, correct? If not, they should be added soon. 
See convo here: https://data-transparency-bfs.slack.com/archives/C0122HUCD17/p1592584034210500?thread_ts=1592406822.150900&cid=C0122HUCD17

Reply:
I'll get these removed

Reply:
Keith is on vacation and I know this has been resolved.